graphs/fleurys_algorithm.py

```diff
@@ -0,0 +1,110 @@
+from collections import defaultdict
+
+
+class Graph:
+
+    def __init__(self, vertices):
+        self.v = vertices
+        self.graph = defaultdict(list)
+
+    def addEdge(self, u, v):
+        """
+        Function to add edges to the graph
+        """
+        self.graph[u].append(v)
+        self.graph[v].append(u)
+
+    def rmvEdge(self, u, v):
+        """
+        Remove u-v edge from the graph
+        """
+        for index, key in enumerate(self.graph[u]):
+            if key == v:
+                self.graph[u].pop(index)
+        for index, key in enumerate(self.graph[v]):
+            if key == u:
+                self.graph[v].pop(index)
+
+    def dfsCount(self, v, visited):
```

Comment:
```suggestion
    def dfs_count(self, v, visited):
```
Real variable names and type hints, please.